Sources/StrongContractResponse/String.swift

```diff
@@ -17,7 +17,7 @@ extension String {
 
         guard let url = defaultPath ??
                 typicalKeys.compactMap(usualCandidates).first else {
-#if !canImport(Vapor)
+            #if canImport(os)
             let errorString =
```

Comment:
These are two different things.  Please explain why you are making this change in a comment at least.